src/benchmark/java/com/commercetools/sync/benchmark/InventorySyncBenchmark.java

```diff
@@ -57,25 +55,23 @@ public static void tearDown() {
 
     @Test
     public void sync_NewInventories_ShouldCreateInventories() throws IOException {
+        // preparation
         final List<InventoryEntryDraft> inventoryEntryDrafts = buildInventoryDrafts(NUMBER_OF_RESOURCE_UNDER_TEST);
-
-
         final InventorySyncOptions inventorySyncOptions = InventorySyncOptionsBuilder.of(CTP_TARGET_CLIENT)
                                                                                      .build();
         final InventorySync inventorySync = new InventorySync(inventorySyncOptions);
 
-
+        // benchmark
         final long beforeSync = System.currentTimeMillis();
         final InventorySyncStatistics inventorySyncStatistics =
             executeBlocking(inventorySync.sync(inventoryEntryDrafts));
         final long totalTime = System.currentTimeMillis() - beforeSync;
 
 
-        // Caclulate sync time and assert on threshold
-        final double diff = calculateDiff(SyncSolutionInfo.LIB_VERSION, INVENTORY_SYNC, CREATES_ONLY, totalTime);
-        assertThat(diff).withFailMessage(format("Diff of benchmark '%e' is longer than expected"
-                            + " threshold of '%d'.", diff, THRESHOLD))
-                        .isLessThanOrEqualTo(THRESHOLD);
+        // assert on threshold (based on history of benchmarks; highest was ~9 seconds)
```

Comment:
It's hard coded here because every single benchmark takes a different time than the other benchmarks. And It's hardcoded because We still don't have results with this new setup. Also it shouldn't dynamically pick up a threshold value because we don't want the execution time to gradually increase by time.